src/core.js

```diff
@@ -157,6 +158,10 @@ const getFrameworkInfo = function (
   return {
     id,
     name,
+    package: {
+      name: detect.npmDependencies[0],
+      version: 'unknown',
```

Comment:
What the reason `version` is set to `'unknown'`?

Reply:
I thought that `undefined` was not a great thing to output to the build logs in the event that there was an issue detecting the version, so I decided on this instead.